mailbox/api/pom.xml

```diff
@@ -68,6 +68,12 @@
             <groupId>org.assertj</groupId>
             <artifactId>assertj-guava</artifactId>
             <scope>test</scope>
+            <exclusions>
```

Comment:
Here the problem is your version "3.0" on assertj-guava. So here you should enforce a version 1.x of assertj-guava rather than excluding assertj-core (exclusions is IMO often a wrong idea)